### ISSUE TITLE
Feature: Accessibility 

### ISSUE THREAD

**Issue by mattgathu** [rust-community/foss-events-planner#11]
**Tasks**
-  write accessibility policy (_early_) - should be before website
- check venue for accessibility (_early_) - before signing on venue
- document venue on website (_early_)
- check alternative venue - restaurant etc (_cfp phase_)
